scripts/linters/third_party_typings_linter.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Lint checks for third party type definitions."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import json
+import os
+import sys
+
+import python_utils
+
+from . import linter_utils
+
+_MESSAGE_TYPE_SUCCESS = 'SUCCESS'
+_MESSAGE_TYPE_FAILED = 'FAILED'
+
+_MANIFEST_JSON_FILE_PATH = os.path.join(os.getcwd(), 'manifest.json')
+_PACKAGE_JSON_FILE_PATH = os.path.join(os.getcwd(), 'package.json')
+
+THIRD_PARTY_LIBS = [
+    {
+        'name': 'Guppy',
+        'manifest.json': 'guppy',
+        'type_defs_file_name': 'guppy-defs-'
+    },
+    {
+        'name': 'Skulpt',
+        'manifest.json': 'skulpt-dist',
+        'type_defs_file_name': 'skulpt-defs-'
+    },
+    {
+        'name': 'Math Expressions',
+        'manifest.json': 'mathExpressions',
+        'type_defs_file_name': 'math-expressions-defs-'
+    },
+    {
+        'name': 'MIDI',
+        'manifest.json': 'midiJs',
+        'type_defs_file_name': 'midi-defs-'
+    },
+    {
+        'name': 'Wavesurfer',
+        'package.json': 'wavesurfer.js',
+        'type_defs_file_name': 'wavesurfer-defs-'
+    },
+    {
+        'name': 'Nerdamer',
+        'package.json': 'nerdamer',
+        'type_defs_file_name': 'nerdamer-defs-'
+    }
+]
+
+
+def check_third_party_libs_type_defs(verbose_mode_enabled):
+    """Checks the type definitions for third party libs
+    are up to date.
+    """
+    stdout = sys.stdout
```

Comment:
This is quite useless to do, you can use `sys.stdout` directly on line 80.

Reply:
Done!